data-serving/scripts/export-data/fields.txt

```diff
@@ -11,7 +11,26 @@ demographics.ethnicity
 demographics.gender
 demographics.nationalities
 demographics.occupation
-events
+events.dateEntry
+events.dateReported
+events.dateLastModified
+events.dateOnset
+events.dateConfirmation
+events.confirmationMethod
+events.dateOfFirstConsult
+events.hospitalized
+events.reasonForHospitalization
+events.dateHospitalization
+events.dateDischargeHospital
+events.intensiveCare
+events.dateAdmissionICU
+events.dateDischargeICU
+events.homeMonitoring
+events.isolated
+events.dateIsolation
+events.outcome
+events.dateDeath
+events.dateRecovered
```

Comment:
Can't comment at line 54 - SGTF was only for the COVID-19 outbreak, can be removed.